sim/simulator_java/src/main/java/cache/SimpleCacheFactory.java

```diff
@@ -13,15 +13,15 @@ public SimpleCacheFactory(){
 	public SimpleCache getCache(Configuration conf){//, SimpleCache nextLevel){
 		SimpleCache cache = null;
 		for(int i = conf.getNumberOfCaches()-1; i >= 0; i--){
-			cache = new OmniCache(conf.getCacheName(i), conf.getCacheSize(i), conf.getBlockSize(i),conf.getAssociativity(i), conf.getWritePolicy(i), conf.getReplacementPolicy(i), cache );
-/*
+			//cache = new OmniCache(conf.getCacheName(i), conf.getCacheSize(i), conf.getBlockSize(i),conf.getAssociativity(i), conf.getWritePolicy(i), conf.getReplacementPolicy(i), cache );
```

Comment:
In the future, let's not save commented-out code. There's `git log -p -S` for finding it in history